irohad/main/server_runner.hpp

```diff
@@ -64,6 +65,7 @@ class ServerRunner {
   std::string serverAddress_;
   bool reuse_;
   std::vector<std::shared_ptr<grpc::Service>> services_;
+  std::string tlsKeypair_;
```

Comment:
We use ``camel_case_`` naming style for internal class members.
```suggestion
  std::string tls_keypair_;
```

Reply:
There is `serverAddress_` literally 4 lines before this one, so I was confused :)

Reply:
Fixed in latest version